### PR TITLE
V10 purchase isolated quotation pyn

### PR DESCRIPTION
**Pull request by phongyanon**
fix error code test from flake8

### CODE REVIEW

File: purchase_isolated_quotation/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Copyright (C) 2012 Ecosoft (<http://www.ecosoft.co.th>)
+#
```
Comment (LoisRForgeFlow) [inline purchase_isolated_quotation/__init__.py]: change to short headers following [OCA template](https://github.com/OCA/maintainer-tools/tree/master/template/module)

File: purchase_isolated_quotation/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+{
+    'name': 'Purchase - Separate Quote and Order',
+    'version': '10.0.0.1.0',
+    'author': 'Ecosoft,Odoo Community Association (OCA)',
+    'category': 'Purchase',
+    'description': """
+This module separate quotation and purchase order
```
Comment (LoisRForgeFlow) [inline purchase_isolated_quotation/__manifest__.py]: Add a README file

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by kittiu** [comment]
@andhit-r commented taht **purchase_order_type** already introduce field name **order_type**. Its better you avoid to use that name. 

@phongyanon I think as it won't be more than 2 type (Order/Quotation), should we use boolean field instead? i.e., isolated_order = True for Purchase Order and False for Quotation.

By doing so, when integrate with other module, i.e., when purchase.order is created by call for bid, the default isolated_order = False and it become Quotation (and we want to start with Quotation as default)

**Comment by phongyanon** [comment]
I've got some exception while run --test-enable, How to fix this?
selection_039

**Comment by kittiu** [comment]
Hi, we are trying to finish this one. The code is done, but the Travis check is not listed (only runbot).
Any clue? Thanks!

File: purchase_isolated_quotation/__init__.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012 Ecosoft (<http://www.ecosoft.co.th>)
```
Comment (kittiu) [inline purchase_isolated_quotation/__init__.py]: @phongyanon I think we can remove the copyright here. Already have the contributor list in README

**Comment by elicoidal** [comment]
Strange indeed: try to trigger the travis directly in travis for this repo. Maybe it will reconnect it again

**Comment by kittiu** [comment]
Don't know how to do it :p.
How about this, @phongyanon can you resubmit your code with new Branch and new PR?

**Comment by elicoidal** [comment]
check this: https://travis-ci.org/OCA/purchase-workflow/pull_requests

**Comment by kittiu** [comment]
Hi, I find the link of the Travis CI here. https://travis-ci.org/OCA/purchase-workflow/builds/238005243
But I have no option to rebuild it. @phongyanon can you?

**Comment by pedrobaeza** [comment]
You have to login with GitHub on Travis and you'll be able to.

**Comment by kittiu** [comment]
Hello again, I want to see if it is ok now, as the travis already green?

File: purchase_isolated_quotation/models/purchase.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        copy=False,
+    )
+    order_id = fields.Many2one(
+        'purchase.order',
+        string='Order Reference',
+        readonly=True,
+        ondelete='restrict',
+        copy=False,
+    )
+    state2 = fields.Selection(
```
Comment (legalsylvain) [inline purchase_isolated_quotation/models/purchase.py]: better to name it quotation_state ?

Comment (phongyanon) [reply purchase_isolated_quotation/models/purchase.py]: i agree with you, it will be meaningful name.

**Pull request closed without merging**